feat: add autoRefresh config field (#2256)

wfrced opened the pull request:
This PR adds functionality requested in #2256. The PR introduces a k9s.autoRefresh config field - the project is very light on CLI args, so I've decided not to introduce another.

Add autoRefresh: true to config test scenarios
Update all instances of NewLiveView() to pass autoRefresh config value to LiveView

derailed reviewed (commented):
@wfrced Thank you Artem for this update! 
I think that might be desirable and was actually the default when this feature was introduced. If memory serves folks did not like it `active` as they could lose their context should a refresh occur while investigation an issue. Hence we've opted to disable it by default. 
I am ok having an option but `autoRefresh` feels way generic since it applies only to live views. Perhaps we should go with `liveViewRefresh` or introduce a config sub section `liveView.autoRefresh: true` or something along these lines. 
Not matter what we do we should also update the docs so users are aware of this feature.
Does this make sense?

wfrced commented:
Thanks for the feedback @derailed. I've renamed the field to `liveViewAutoRefresh` and added it to the docs.
A config subsection for `liveView` entities seems to be overkill, since that would be the only reasonably configurable field.

derailed commented on internal/view/live_view.go:
```diff
@@ -37,7 +37,7 @@ type LiveView struct {
 
 // NewLiveView returns a live viewer.
-func NewLiveView(app *App, title string, m model.ResourceViewer) *LiveView {
+func NewLiveView(app *App, title string, m model.ResourceViewer, autoRefresh bool) *LiveView {
```
I don't think we need to change the signature here. Since the app is passed in we can pull the liveview autorefresh config directly in the view ctor.

wfrced replied:
Right, sure!

derailed reviewed (changes requested):
@wfrced Thank you Artem for the updates, just a small pick...

derailed reviewed (approved):
@wfrced Thank you for the updates Artem!!
